sdk/src/client/node_api/core/routes.rs

```diff
@@ -222,7 +222,7 @@ impl ClientInner {
             .await
     }
 
-    /// Returns the block that was included in the ledger for a given transaction ID, as object.
+    /// Returns the earliest block containing the transaction that get confirmed, as object.
```

Comment:
awkward phrasing

Reply:
It's from the TIP, do you have a better suggestion?

Reply:
```rust
/// Returns the earliest confirmed block containing the transaction with the given ID.
```

Reply:
Thanks, used that